lib/ansible/plugins/connection/network_cli.py

```diff
@@ -84,6 +88,9 @@ def _connect(self):
 
         display.display('ssh connection done, setting terminal', log_only=True)
 
+        self._shell = self.ssh.invoke_shell()
```

Comment:
`open_shell` is a dummy command to start `ansible-connection`
`ansible-connection` is refactored to remove this dependency.